generators/javascript.js

```diff
@@ -321,3 +321,5 @@ Blockly.JavaScript.getAdjusted = function(block, atId, opt_delta, opt_negate,
   }
   return at;
 };
+
+exports = JavaScript;
```

Comment:
is this supposed to be a default export at this stage?

Reply:
Yes, currently doing default exports.